martin/src/config/file/pg/config_function.rs

```diff
@@ -5,7 +5,7 @@ use tilejson::{Bounds, TileJSON};
 
 use super::config::PgInfo;
 use crate::config::file::UnrecognizedValues;
-use crate::pg::utils::patch_json;
+use crate::config::file::pg::utils::patch_json;
```

Comment:
kinda weird that we have `config/file/pg`  -- i would have thought that `/file` is for pmtiles/mbtiles/... other file-based sources, whereas `config/pg` is explicit for pg. Not a biggie, just a bit confusing